controllers/updateservice_controller.go

```diff
@@ -350,6 +351,38 @@ func (r *UpdateServiceReconciler) ensureTrustedClusterCA(ctx context.Context, re
 	return err
 }
 
+func (r *UpdateServiceReconciler) ensureGraphDataSHA(ctx context.Context, reqLogger logr.Logger, instance *cv1.UpdateService,
+	resources *kubeResources) (string, error){
+
+	pod := resources.graphDataPodContainer
```

Comment:
can we add a guard here so that we `return` quickly for by-tag graph-data pullspecs?  When the pullspec is already by-digest, launching a Pod for a tag-to-digest lookup is unnecessary overhead.

Reply:
added